routers/api/v1/org/action.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package org
+
+import (
+	"net/http"
+
+	"code.gitea.io/gitea/models/secret"
+	"code.gitea.io/gitea/modules/context"
+	api "code.gitea.io/gitea/modules/structs"
+	"code.gitea.io/gitea/routers/api/v1/utils"
+)
+
+// ListActionsSecrets list an organization's actions secrets
+func ListActionsSecrets(ctx *context.APIContext) {
+	// swagger:operation GET /orgs/{org}/actions/secrets organization orgListActionsSecrets
```

Comment:
Erm… User secrets exist too already.

Reply:
That and we were not sure if we keep them action-only.
I think it is better to omit the `actions/` part from the URL.

Reply:
Or do you explicitly only want to return the action secrets here?

Reply:
I'm sorry if my previous response was unclear. Please provide more details or elaborate on what you want me to explain. Thank you.

Reply:
1. We try to be compatible with GitHub API where possible, so far so good
2. I think this is the wrong URL for that route, I suggest `/orgs/{org}/secrets`/`/users/{user}/secrets` instead

That leaves the question what to do with the missing URL.
I suggest adding three routes in this PR: The two mentioned above, and the one you already implemented.
However, as the third should only be used by existing scripts and nothing else, I recommend marking this route as `// deprecated: true` with a comment that it is only intended for GitHub API compatibility and that our routes are the other two above instead.

Reply:
As I said in https://github.com/go-gitea/gitea/pull/24200#issuecomment-1514407021 , we can provide other API routes for more purposes but share the data table.

So LGTM.

Reply:
@delvh @wolfogre 

replace the route /orgs/{org}/actions/secrets with /orgs/{org}/secrets